build.yaml

```diff
@@ -4816,9 +4816,9 @@ steps:
       set -ex
       tar xvf /io/wheel-container.tar
       gcloud auth activate-service-account --key-file=/gsa-key/key.json
-      HAIL_WHEEL_URL=gs://cpg-hail-ci/wheels/hail-$(cat /io/hail_pip_version)-py3-none-any.whl
+      HAIL_WHEEL_FILE=hail-$(cat /io/hail_pip_version)-py3-none-any.whl
       python3 -m hailtop.aiotools.copy 'null' '[
-      {"from": "hail-*-py3-none-any.whl", "to": "'${HAIL_WHEEL_URL}'"}]'
+      {"from": "'$HAIL_WHEEL_FILE'", "to": "gs://cpg-hail-ci/wheels/'$HAIL_WHEEL_FILE'"}]'
```

Comment:
This is one of my least favourite sections of bash, putting in env variables with single quotes.

Reply:
Yeah, quoting in general is such a nightmare. Basically one just hopes that variables never contain spaces or quotes themselves... ;)